x-pack/plugins/security/server/saved_objects/secure_saved_objects_client_wrapper.ts

```diff
@@ -199,20 +199,22 @@ export class SecureSavedObjectsClientWrapper implements SavedObjectsClientContra
     objects: Array<SavedObjectsBulkUpdateObject<T>> = [],
     options: SavedObjectsBaseOptions = {}
   ) {
-    await this.ensureAuthorized(
-      this.getUniqueObjectTypes(objects),
-      'bulk_update',
-      options && options.namespace,
-      { objects, options }
-    );
+    const objectNamespaces = objects
+      .filter(({ namespace }) => namespace !== undefined)
```

Comment:
question: is there a need to filter these out here? `options?.namespace` below could also be undefined, but we're still including that (as we should). We end up looping over all namespaces to convert and deduplicate later on anyway, so this `filter` feels unnecessary

Reply:
An `undefined` object namespace is not the Default namespace, it is the absence of a namespace. (unlike options namespace, where `undefined` is equivalent to the Default namespace). So we have to filter these out.

Otherwise, this:
```ts
bulkGet({id: 'foo', type: 'bar'}, { namespace: 'spacex' });
```
would check if the user is authorized for the Default space and the spacex space.

Reply:
Ah of course! Ignore me, and thanks for clarifying